### PR TITLE
Display form responses for the anim when clicking on display forms bu…

### PR DESCRIPTION
**Pull request by remiriv**
…tton

## Proposed changes

Ticket: https://ogre-d4g.atlassian.net/browse/D4G-189

- Add button in anim interface to display forms responses by players
- Cancel / Validate buttons on the popup (form edition in another ticket) 

---

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

---

What types of changes does your code introduce to ogre ?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe):

## Checklist

---

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

offseason_ogre/blob/master/CONTRIBUTING.md) doc

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation and modify the README.md (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] If the feature changes current behavior, reasons why your solution is better.

### CODE REVIEW

**Review by louis-sanna-dev — APPROVED** [review]

File: packages/client/src/modules/administration/Games/Game/FormVerification.tsx [hunk 1]
```diff
@@ -0,0 +1,543 @@
...
+  return (
+    <VerificationContainer show={openFormValidation}>
+      <DataGridBox sx={{ backgroundColor: "white", color: "black" }}>
+        <DataGrid
+          sx={{ textAlign: "center" }}
+          rows={rows}
+          columns={buildColumns()}
+          disableSelectionOnClick
+          experimentalFeatures={{ newEditingApi: true }}
+          //   processRowUpdate={(newRow, oldRow) => {
```
Comment (Baboo7) [inline packages/client/src/modules/administration/Games/Game/FormVerification.tsx]: Commentaire 🔫  pew pew

File: packages/client/src/modules/administration/Games/Game/FormVerification.tsx [hunk 2]
```diff
@@ -0,0 +1,543 @@
...
+    {
+      field: "eatingDogNumber",
+      headerName: "Nb chiens",
+      flex: 1,
+      minWidth: 90,
+      headerAlign: "center",
+      align: "center",
+    },
+    {
+      field: "eatingHorse",
```
Comment (Baboo7) [inline packages/client/src/modules/administration/Games/Game/FormVerification.tsx:488]: Savage

File: packages/client/src/modules/persona/consumption/utils.ts [hunk 3]
```diff
@@ -65,11 +65,11 @@ export const getCarAgeCoeff = (carAge: string) => {
   if (carAge === carAges.MOINS_5) {
     return 3;
   } else if (carAge === carAges.CINQ_DIX) {
-    return 0.75;
+    return 2;
```
Comment (Baboo7) [inline packages/client/src/modules/persona/consumption/utils.ts]: [Non important] Quand on a affaire à un mapping je trouve plus lisible d'avoir un objet de config :

```ts
const carAgeToCoeff = {
  [carAges.MOINS_5]: 3,
  [carAges.CINQ_DIX]: 2,
  [carAges.DIX_QUINZE]: 1,
  [carAges.PLUS_15]: 1,
}

export const getCarAgeCoeff = getCoeff("car age", carAgeToCoeff);

const getCoeff = (configName, mapping) => (key: string) => {
  const coeff = mapping[key];

  if (coeff == null) {
    throw new Error(`Invalid profile value for ${configName}: ${key}`);
  }

  return coeff;
};
```

Ça permet d'enrichir facilement le système en ajoutant un élément dans le mapping sans avoir à toucher à la logique. Et c'est transposable à d'autres mapping comme celui du temps de douche au dessus par exemple

File: packages/client/src/modules/administration/Games/Game/FormVerification.tsx (hunk 2 above)
Comment (remiriv) [reply packages/client/src/modules/administration/Games/Game/FormVerification.tsx:488]: Findus en PLS

File: packages/client/src/modules/administration/Games/Game/FormVerification.tsx (hunk 1 above)
Comment (remiriv) [reply packages/client/src/modules/administration/Games/Game/FormVerification.tsx]: 😬😬

File: packages/client/src/modules/persona/consumption/utils.ts (hunk 3 above)
Comment (remiriv) [reply packages/client/src/modules/persona/consumption/utils.ts]: Effectivement beaucoup plus clean

**Pull request merged**